packages/hoppscotch-common/src/helpers/import-export/import/hoppEnv.ts

```diff
@@ -17,15 +17,33 @@ const hoppEnvSchema = z.object({
   ),
 })
 
-export const hoppEnvImporter = (content: string) => {
-  const parsedContent = safeParseJSON(content, true)
+type HoppEnv = z.infer<typeof hoppEnvSchema>
 
-  // parse json from the environments string
-  if (O.isNone(parsedContent)) {
+export const hoppEnvImporter = (contents: string[]) => {
+  const parsedContents = contents.map((str) => safeParseJSON(str, true))
+
+  if (parsedContents.some((parsed) => O.isNone(parsed))) {
     return TE.left(IMPORTER_INVALID_FILE_FORMAT)
   }
 
-  const validationResult = z.array(hoppEnvSchema).safeParse(parsedContent.value)
+  const parsedValues = parsedContents.flatMap((content) => {
+    const unwrappedContent = O.toNullable(content) as HoppEnv[] | null
+
+    if (unwrappedContent) {
+      return unwrappedContent.map((contentEntry) => {
+        return {
+          ...contentEntry,
+          variables: contentEntry.variables?.map((valueEntry) => ({
+            ...valueEntry,
+            value: String(valueEntry.value),
```

Comment:
The safeguard for `variables` prevents exceptions while attempting to import environments from other platforms, say, Postman, that use `values` to refer to variable entries. Also, all the values are stringified before checking against the schema.